.github/dependabot.yml

```
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: /web
+    schedule:
+      interval: weekly
```

```suggestion
      interval: monthly
```
I would suggest to lower the interval to `monthly`, as Dify itself is not a upstream package with lower demand for dependency updates especially for the frontend module.